recipe/meta.yaml

```diff
@@ -82,6 +87,15 @@ requirements:
 # Remove it with a new scikit-image release >=0.25.0.
 {% set tests_to_skip = tests_to_skip + " or test_thresholds_dask_compatibility" %}
 
+# see https://github.com/scipy/scipy/issues/20200 and https://github.com/scipy/scipy/issues/7408 
+{% set tests_to_skip = tests_to_skip + " or test_2d_bf[float16] or test_2d_cg[float16] or test_2d_cg_mg[float16] or test_2d_cg_j[float16]" %}
```

Comment:
Was this failing on all architecture?

Reply:
Yes, all platforms

Reply:
I've tested on osx. Let's see if other platforms fail

Reply:
Yes, I can confirm that it failed on all platforms. I left a comment